### ISSUE TITLE
NASNetLarge Keras Model - accuracy is very very low

### ISSUE THREAD

**Issue by suresh-s** [tensorflow/tensorflow#43730]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):  No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: Issue happening on the Server while training
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): tensorflow-gpu==2.0.0
- Python version: 3.6.9
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: CUDA 10.0 and cuDNN 7.6.5
- GPU model and memory: 2 GPU Tesla K80 and total memory 22 GB

**Describe the current behavior**
Keras version 2.3.0
Tensorflow-gpu version 2.0.0

Dataset: Imagenet cars dataset
Preprocessing : Resizing images to 331 * 331 pixel using custom code and saving as new files, no other preprocessing

I tested the NASNetLarge Keras model with parameters as below

```
from keras.applications.nasnet import NASNetLarge 
model=NASNetLarge(input_shape =  (331, 331, 3),include_top = False,  weights = 'imagenet')
model.trainable = False

```
and created combined model using Sequential
Training Data and valid data processed through flow_from_directory and fed fit_generator 

Loss : approx 5.0
val_acc: approx 0.00813

**Comment by amahendrakar** [comment]
@suresh-s,
In order to expedite the trouble-shooting process, could you please provide the complete code to reproduce the issue reported here and the dataset you are using. 

Also, please update TensorFlow to the latest version v2.3 and check if you are facing the same issue. Thanks!
